### PR TITLE
Postal address on usersheet

### PR DESCRIPTION
**Pull request by marcelb98**
To send the usersheets by post this PR adds the postal address of the users to the usersheets. Therefore the usersheet has been slightly restructured.

The usersheet now looks like this:
user_sheet

### CODE REVIEW

File: pycroft/helpers/printing/__init__.py [hunk 1]
```diff
@@ -84,39 +84,40 @@ def generate_user_sheet(user, user_id, plain_password, generation_purpose=''):
...
-                  'visit us during our office hours. You can find contact '
-                  'information below on this page.',
-                  style['BodyText']))
+                  style['BodyText'])
+
+    return_notice = Paragraph(
+        '''<font size="9pt">Nicht nachsenden!</font>''',
+        style['Normal'])
+    sender = Paragraph(
+        '''<font size="9pt">AG DSN • BuS • Wundtstraße 5 • 01217 Dresden</font>''',
```
Comment (lukasjuhrich) [inline pycroft/helpers/printing/__init__.py]: Please refrain from abbreviations like `BuS` whose meaning is only known to active members of the AG DSN or related persons.

File: pycroft/model/address.py [hunk 2]
```diff
@@ -36,3 +36,12 @@ def __str__(self):
...
         return ", ".join(items)
+
+    def postal(self):
+        items: List[str] = [f"{self.street} {self.number}\n{self.addition}" if self.addition
+                            else f"{self.street} {self.number}", f"{self.zip_code} {self.city}"]
+        if self.state:
+            items.append(f"{self.state}")
+        if self.country and self.country != DEFAULT_COUNTRY:
+            items.append(f"{self.country}")
+        return "\n".join(items)
```
[the remark this answers is not in the document]
Comment (lukasjuhrich) [reply pycroft/model/address.py]: (if I weren't on Deadalus right now I would fix that myself)

[the remark this answers is not in the document]
Comment (lukasjuhrich) [reply pycroft/model/address.py]: FYI: Also note #351

File: scripts/server_run.py [hunk 3]
```diff
@@ -19,7 +19,7 @@
 from pycroft.model.session import set_scoped_session
 from scripts.schema import AlembicHelper, SchemaStrategist
 from web import make_app
-from .connection import try_create_connection, get_connection_string
+from scripts.connection import try_create_connection, get_connection_string
```
[the remark this answers is not in the document]
Comment (marcelb98) [reply scripts/server_run.py]: Without this change the import fails and the dev-app container is not starting

[the remark this answers is not in the document]
Comment (lukasjuhrich) [reply scripts/server_run.py]: then we'll keep the fix and investigate another time.

**Pull request merged**